lib/experimental/Navigation/Header/PageHeader/index.tsx

```diff
@@ -74,21 +83,31 @@ export function PageHeader({
           )}
         </div>
       </div>
-      {actions && (
-        <div className="flex items-center gap-2">
-          {actions.map((action, index) => (
-            <Button
-              hideLabel
-              round
-              key={index}
-              variant="outline"
-              onClick={action.onClick}
-              label={action.label}
-              icon={action.icon}
-            />
-          ))}
-        </div>
-      )}
+      <div className="flex items-center">
+        {!hasNavigation && hasStatus && (
+          <div className="pe-3">
+            <StatusTag text={statusTag.text} variant={statusTag.variant} />
+          </div>
+        )}
+        {hasStatus && hasActions && (
+          <div className="right-0 h-4 w-px bg-f1-border-secondary"></div>
```

Comment:
can you take a look if we have an existing divider component?

Reply:
It existed but @josepjaume told me we have removed it to make use of borders

Reply:
The problem with the old divider is that it was used once and in the wrong place (that should have been a border), so I removed without knowing you were using it in this PR :)

I honestly think this is fine, as a divider is not much else than a bunch of tokens. I would think about it later if we find more use cases.

Reply:
> I honestly think this is fine, as a divider is not much else than a bunch of tokens. I would think about it later if we find more use cases.

A bunch of tokens everyone does from scratch and slightly differently. Let's proceed here anyways

Reply:
I understand, yes. The thing is that it looks like this specific divider might be just scoped to a "Toolbar" no? We probably need to define that component?